### ISSUE TITLE
GITHUB Repository Problem 

### ISSUE THREAD

**Issue by lyrl** [OregonCore/OregonCore#1220]
I checkout a fresh copy of github repository on my mac book.

i do not any change.

when i type `git status` show.

i try to type `git reset --hard` is not change.

```
    modified:   src/scripts/EasternKingdoms/SunwellPlateau/boss_felmyst.cpp
    modified:   src/scripts/EasternKingdoms/SunwellPlateau/boss_kiljaeden.cpp
    modified:   src/scripts/EasternKingdoms/SunwellPlateau/boss_muru.cpp
    modified:   src/scripts/EasternKingdoms/ZulAman/boss_akilzon.cpp
    modified:   src/scripts/EasternKingdoms/undercity.cpp
    modified:   src/scripts/Kalimdor/ashenvale.cpp
    modified:   src/scripts/Kalimdor/azuremyst_isle.cpp
    modified:   src/scripts/Kalimdor/darkshore.cpp
    modified:   src/scripts/Kalimdor/desolace.cpp
    modified:   src/scripts/Kalimdor/silithus.cpp
    modified:   src/scripts/Outland/Auchindoun/shadow_labyrinth/boss_ambassador_hellmaw.cpp
    modified:   src/scripts/Outland/Auchindoun/shadow_labyrinth/instance_shadow_labyrinth.cpp
    modified:   src/scripts/Outland/TempestKeep/the_eye/boss_kaelthas.cpp
    modified:   src/scripts/Outland/blades_edge_mountains.cpp
    modified:   src/scripts/Outland/hellfire_peninsula.cpp
    modified:   src/scripts/Outland/zangarmarsh.cpp
    modified:   src/scripts/World/areatrigger_scripts.cpp
    modified:   src/scripts/World/item_scripts.cpp
    modified:   src/scripts/World/npcs_special.cpp
    modified:   src/shared/Database/DBCFileLoader.cpp
    modified:   src/shared/Database/Field.h
    modified:   src/tools/movements_extractor/CMakeLists.txt
    modified:   src/tools/movements_extractor/Extract/mmap_extract.py
    modified:   src/tools/movements_extractor/Info/offmesh_example.txt
    modified:   src/tools/movements_extractor/Info/readme

no changes added to commit (use "git add" and/or "git commit -a")
```

**Comment by LordUsagi** [comment]
This could be a problem with line endings

**Comment by lyrl** [comment]
Many File Still Is CRLF Ending On Repository.

```

    modified:   src/game/HomeMovementGenerator.cpp
    modified:   src/game/IdleMovementGenerator.cpp
    modified:   src/game/InstanceData.h
    modified:   src/game/InstanceSaveMgr.cpp
    modified:   src/game/LootHandler.cpp
    modified:   src/game/LootMgr.h
    modified:   src/game/MapScripts.cpp
    modified:   src/game/MotionMaster.cpp
    modified:   src/game/MoveSpline.cpp
    modified:   src/game/MoveSpline.h
    modified:   src/game/MoveSplineFlag.h
    modified:   src/game/MoveSplineInit.cpp
    modified:   src/game/MoveSplineInit.h
    modified:   src/game/MoveSplineInitArgs.h
    modified:   src/game/ObjectAccessor.h
    modified:   src/game/ObjectGridLoader.cpp
    modified:   src/game/ObjectGuid.h
    modified:   src/game/ObjectMgr.cpp
    modified:   src/game/ObjectMgr.h
    modified:   src/game/PathFinder.h
    modified:   src/game/Pet.h
    modified:   src/game/PetAI.cpp
    modified:   src/game/PetHandler.cpp
    modified:   src/game/PointMovementGenerator.cpp
    modified:   src/game/PointMovementGenerator.h
    modified:   src/game/QuestDef.h
    modified:   src/game/ScriptMgr.cpp
    modified:   src/game/ScriptMgr.h
    modified:   src/game/ScriptedCreature.cpp
    modified:   src/game/ScriptedEscortAI.cpp
    modified:   src/game/ScriptedFollowerAI.cpp
    modified:   src/game/ScriptedInstance.h
    modified:   src/game/SmartAI.cpp
    modified:   src/game/SmartAI.h
    modified:   src/game/SmartScript.cpp
    modified:   src/game/SmartScript.h
    modified:   src/game/SmartScriptMgr.cpp
    modified:   src/game/SmartScriptMgr.h
    modified:   src/game/Spell.h
    modified:   src/game/SpellMgr.h
    modified:   src/game/TargetedMovementGenerator.cpp
    modified:   src/game/TargetedMovementGenerator.h
    modified:   src/game/TemporarySummon.h
    modified:   src/game/UnitAI.h
    modified:   src/game/WaypointMovementGenerator.cpp
    modified:   src/game/Weather.cpp
    modified:   src/game/packet_builder.cpp
    modified:   src/game/packet_builder.h
    modified:   src/game/spline.cpp
    modified:   src/game/spline.impl.h
    modified:   src/game/typedefs.h
    modified:   src/game/util.cpp
    modified:   src/scripts/EasternKingdoms/Karazhan/boss_shade_of_aran.cpp
    modified:   src/scripts/EasternKingdoms/Karazhan/instance_karazhan.cpp
    modified:   src/scripts/EasternKingdoms/MoltenCore/boss_garr.cpp
    modified:   src/scripts/EasternKingdoms/Naxxramas/boss_kelthuzad.cpp
    modified:   src/scripts/EasternKingdoms/ScarletMonastery/boss_mograine_and_whitemane.cpp
    modified:   src/scripts/EasternKingdoms/Stratholme/boss_baron_rivendare.cpp
    modified:   src/scripts/EasternKingdoms/SunwellPlateau/boss_brutallus.cpp
    modified:   src/scripts/EasternKingdoms/SunwellPlateau/boss_felmyst.cpp
    modified:   src/scripts/EasternKingdoms/SunwellPlateau/boss_kiljaeden.cpp
    modified:   src/scripts/EasternKingdoms/SunwellPlateau/boss_muru.cpp
    modified:   src/scripts/EasternKingdoms/ZulAman/boss_akilzon.cpp
    modified:   src/scripts/EasternKingdoms/undercity.cpp
    modified:   src/scripts/Kalimdor/ashenvale.cpp
    modified:   src/scripts/Kalimdor/azuremyst_isle.cpp
    modified:   src/scripts/Kalimdor/darkshore.cpp
    modified:   src/scripts/Kalimdor/desolace.cpp
    modified:   src/scripts/Kalimdor/silithus.cpp
    modified:   src/scripts/Outland/Auchindoun/shadow_labyrinth/boss_ambassador_hellmaw.cpp
    modified:   src/scripts/Outland/Auchindoun/shadow_labyrinth/instance_shadow_labyrinth.cpp
    modified:   src/scripts/Outland/TempestKeep/the_eye/boss_kaelthas.cpp
    modified:   src/scripts/Outland/blades_edge_mountains.cpp
    modified:   src/scripts/Outland/hellfire_peninsula.cpp
    modified:   src/scripts/Outland/zangarmarsh.cpp
    modified:   src/scripts/World/areatrigger_scripts.cpp
    modified:   src/scripts/World/item_scripts.cpp
    modified:   src/scripts/World/npcs_special.cpp
    modified:   src/shared/Database/DBCFileLoader.cpp
    modified:   src/shared/Database/Field.h
    modified:   src/tools/movements_extractor/CMakeLists.txt
    modified:   src/tools/movements_extractor/Extract/mmap_extract.py
    modified:   src/tools/movements_extractor/Info/offmesh_example.txt
    modified:   src/tools/movements_extractor/Info/readme
```

**Comment by LordUsagi** [comment]
Fixed in 0420cfa5fa67
